server/handler/handler.go

```diff
@@ -0,0 +1,49 @@
+package handler
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/mattermost/release-bot/config"
+	"github.com/mattermost/release-bot/store"
+	"github.com/mattermost/release-bot/version"
+	"github.com/palantir/go-githubapp/githubapp"
+	"github.com/rcrowley/go-metrics"
+)
+
+const (
+	githubHandlerDefaultRoute  string = "/hook"
+	healthHandlerDefaultRoute  string = "/healthz"
+	createCheckRunDefaultRoute string = "/checkrun/create"
+	finishCheckRunDefaultRoute string = "/checkrun/update"
+)
+
+func Register(config *config.Config) {
+	version := version.Full()
+	metricsRegistry := metrics.DefaultRegistry
+	eventContextStore := store.NewEventContextStore()
+
+	cc, err := githubapp.NewDefaultCachingClientCreator(
+		config.Github,
+		githubapp.WithClientUserAgent(fmt.Sprintf("%s/%s", version.Name, version.Version)),
+		githubapp.WithClientTimeout(10*time.Second),
+		githubapp.WithClientMiddleware(
+			githubapp.ClientMetrics(metricsRegistry),
+		),
+	)
+
+	if err != nil {
+		panic(err)
```

Comment:
better to not panic in such cases but just exit with an error message.

Reply:
Panic replaced with `os.Exit`..